public/server/db.json

```diff
@@ -169,183 +175,66 @@
-  },
-
-  "fyi": {
-    "definition": "For Your Information"
-  },
-
-  "iykyk": {
-    "definition": "If You Know You Know"
-  },
-
-  "ifg": {
-    "definition": "I Feel Good"
-  },
-
-  "igtg": {
-    "definition": "I've Got To Go"
-  },
-
-  "icymi": {
-    "definition": "In Case You Missed It"
-  },
-
-  "ikr": {
-    "definition": "I Know, Right?"
-  },
-
-  "kiss": {
-    "definition": "Keep It Simple Stupid",
-    "alternatives": "Keep It Super Simple, Keep It Short And Simple"
-  },
-
-  "hmu": {
-    "definition": "Hit Me Up"
-  },
-
-  "lol": {
-    "definition": "Laugh Out Loud"
-  },
-
-  "stfu": {
-    "definition": "Shut The Fuck Up"
-  },
-
-  "ngl": {
-    "definition": "Not Gonna Lie"
-  },
-
-  "hru": {
-    "definition": "How Are You"
-  },
-
-  "srly": {
-    "definition": "Seriously"
-  },
-
-  "bff": {
-    "definition": "Best Friend Forever"
+  "smh": {
+    "definition": "Shaking My Head"
   },
-
-  "tl": {
-    "definition": "Timeline"
+  
+  "pov": {
+    "definition": "Point Of View"
   },
-
-  "cl": {
-    "definition": "Contact List"
+  
+  "tbf": {
+    "definition": "To Be Frank",
+    "alternatives": "To Be Fair"
   },
-
-  "nvm": {
-    "definition": "Never Mind"
+  
+  "fgs": {
+    "definition": "For God's sake"
   },
 
-  "wyd": {
-    "definition": "What You Doing"
+  "tysm": {
+    "definition": "Thank You So Much"
   },
 
-  "lmk": {
-    "definition": "Let Me Know"
+  "ty": {
+    "definition": "Thank You"
   },
 
-  "idc": {
-    "definition": "I Don't Care"
-  },
+  "syl": {
+    "definition": "See You Later"
+  }, 
 
-  "til": {
-    "definition": "Today I Learn",
-    "alternatives": "Today I Learned"
-  },
+  "hbty": {
+  	"definition": "Happy Birthday To You"
+  }, 
 
-  "brb": {
-    "definition": "Be Right Back"
-  },
+  "hbu": {
+  	"definition": "How About You"
+  }, 
 
-  "byob": {
-    "definition": "Bring Your Own Beer",
-    "alternatives": "Be Your Own Boss"
-  },
+  "wth": {
+  	"definition": "What The Hell"
+  }, 
 
-  "tldr": {
-    "definition": "Too Long, Didn't Read"
-  },
+  "imy": {
+  	"definition": "I Miss You"
+  }, 
 
-  "ttyl": {
-    "definition": "Talk To You Later"
-  },
+  "ilu": {
+  	"definition": "I Love You",
+  	"definition": "I Like You"
+  }, 
 
-  "tc": {
-    "definition": "Take Care"
-  },
+  "sys": {
+  	"definition": "See You Soon"
+  }, 
 
-  "rofl": {
-    "definition": "Rolling On The Floor, Laughing"
-  },
+  "ik": {
+  	"definition": "I Know"
+  }, 
 
-  "ootd": {
-    "definition": "Outfit Of The Day"
-  },
-  
-  "goat": {
-    "definition": "Greatest Of All Time"
-  },
-  
-  "tfw": {
-    "definition": "The Feeling When"
-  },
-  
-  "smh": {
-    "definition": "Shaking My Head"
-  },
-  
-  "pov": {
-    "definition": "Point Of View"
-  },
-  
-  "tbf": {
-    "definition": "To Be Frank",
-    "alternatives": "To Be Fair"
-  },
-  
-  "fgs": {
-    "definition": "For God's sake"
+  "dygm": {
+  	"definition": "Did You Get Me"
   }
-  
+
```

Comment:
Looks like there are some definitions not in here that are in `main` branch. To resolve that, please fix the merge conflict to bring main version of `db.json` into your version. Let me know if you need assistance or run into issues, I got you.